com.unity.shadergraph/Editor/ShaderFoundry/Samples/SimpleSample1.cs

```diff
@@ -12,13 +12,14 @@ internal static void RunSimpleTest()
             // To build a shader, you need to provide a container and a target.
             // The shader code will be in the shader builder passed in.
             var target = SimpleSampleBuilder.GetTarget();
+
             var container = new ShaderContainer();
-            var shaderBuilder = new ShaderBuilder();
             SimpleSampleBuilder.BuildCommonTypes(container);
-            SimpleSampleBuilder.Build(container, target, ShaderName, BuildSample, shaderBuilder);
 
-            var code = shaderBuilder.ToString();
-            DisplayTestResult(ShaderName, code);
+            var primaryShaderID = string.Empty;
```

Comment:
It looks like the Build call has the final parameter default initialized.  Do you need this empty string, or would the effect be the same if you didn't supply the parameter?

Reply:
Yes, it should be equivalent to not supplying the parameter.   I was adding it to show you could supply it (as this is a bit of sample code).. and to call out that it is only generating the primary shader here.

Reply:
Cool.  Just checking.